screens/DaterSetup.tsx

```
@@ -0,0 +1,380 @@
+    return (
+        <ScrollView style={commonStyles.container} contentContainerStyle={commonStyles.contentContainer}>
+            <AppBar />
+
+            <ThemedText style={commonStyles.title}>
+                Welcome!
+            </ThemedText>
+            
+            {/* Step 1 */}
+            <ThemedView style={commonStyles.section}>
+                <ThemedText style={commonStyles.sectionTitle}>
+                    Step 1: Tell us why you're here
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)